RELEASE_PROCESS_OPENSEARCH.md

```diff
@@ -0,0 +1,313 @@
+All components (which are ready after completion of version increment)
+```
+
+##### OpenSearch Dashboards
+```
+OpenSearch Dashboards
+All components (which are ready after completion of version increment)
+```
+
+### Release Candidate
+
+The automated build workflows will create release candidates with the finalized commit and publish the docker images and associated distribution artifacts for testing. Now once all the version increment PR’s are completed and all the components are part of the input manifest, now its time to generate the RC.
+
+
+#### Sample Build details:
+
+| OpenSearch | OpenSearch Dashboards |
+|----------|----------|
+| [build_7858](https://build.ci.opensearch.org/job/distribution-build-opensearch/7848/console) | [build_6126](https://build.ci.opensearch.org/job/distribution-build-opensearch-dashboards/6126/) |
+
+The distribution build number denotes the RC, now with the above example finalized RC’s are `OS: 7848`, `OSD: 6126`.
+
+#### Integ Test Tar:
+
+| Integ Test Tar | x64 | arm64 |
+|----------|----------|----------|
+| OpenSearch  | [x64](https://build.ci.opensearch.org/job/integ-test/4906/console)  | [arm64](https://build.ci.opensearch.org/job/integ-test/4897/console)   |
+| OpenSearch Dashboards  | [x64](https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/3531/console)  | [arm64](https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/3530/console)   |
+
+#### Integ Test RPM:
+
+| Integ Test RPM | x64 | arm64 |
+|----------|----------|----------|
+| OpenSearch  | [x64](https://build.ci.opensearch.org/job/integ-test/4908/console)  | [arm64](https://build.ci.opensearch.org/job/integ-test/4907/console)   |
+| OpenSearch Dashboards  | [x64](https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/3532/console)  | [arm64](https://build.ci.opensearch.org/job/integ-test-opensearch-dashboards/3533/console)   |
+
+#### Docker Build and Scan: 
+
+| Docker | build | scan |
+|----------|----------|----------|
+| OpenSearch  | [Build](https://build.ci.opensearch.org/job/docker-build/3371/)  | [Scan](https://build.ci.opensearch.org/job/docker-scan/1558/artifact/scan_docker_image.txt)   |
+| OpenSearch Dashboards  | [Build](https://build.ci.opensearch.org/job/docker-build/3370/)  | [Scan](https://build.ci.opensearch.org/job/docker-scan/1557/artifact/scan_docker_image.txt)   |
+
+
+#### Docker freeze RC: 
+This to ensure that [check-for-build.jenkinsfile](https://github.com/opensearch-project/opensearch-build/blob/main/jenkins/check-for-build.jenkinsfile) wont re-build periodically and override the docker, the RC docker is created with build number.
+
+| Docker Freeze | copy |
+|----------|----------|
+| OpenSearch  | [docker-copy](https://build.ci.opensearch.org/job/docker-copy/466/console)  | 
+| OpenSearch Dashboards  | [docker-copy](https://build.ci.opensearch.org/job/docker-copy/467/console)  | 
```

Comment:
These are kinda duplicate of each other to some degree.
In fact some times we prefer to run a copy after the build to ensure the build is complete without issues, before copying the images.

Reply:
Updated with some more details peter, please check in my latest commit.